client/src/components/main/Nav/FusionTabs.tsx

```diff
@@ -0,0 +1,30 @@
+import { Box, Typography, makeStyles } from "@material-ui/core";
+import React, { useContext } from "react";
+import { FusionContext } from "../../../global/contexts/FusionContext";
+
+const useStyles = makeStyles(() => ({
+  fusionType: {
+    marginRight: '0.5em',
+    color: 'white',
+    fontWeight: 'normal',
+  },
+}));
+
+export default function FusionTabs(): React.ReactElement {
+  const classes = useStyles()
+  const { fusion, setFusion } = useContext(FusionContext);
+
+  console.log(fusion)
+
+  const fusionType = fusion.type === 'CategoricalFusion' ? 'Categorical Fusion' : 'Assayed Fusion'
+  const switchToType = fusion.type === 'CategoricalFusion' ? 'Assayed Fusion' : 'Categorical Fusion'
+
+  return (
+    <div>
+      <Box display='flex'>
+        <h2 className={classes.fusionType}>{fusionType}</h2>
+        <Typography color='inherit' style={{ cursor: 'pointer', margin: 'auto' }} onClick={() => { setFusion({ ...fusion, type: switchToType.replace(/\s/g, '') }) }}>(Switch to {switchToType} tool?)</Typography>
```

Comment:
This should be more clear that it's a button. When I first saw it, I thought it was just plaintext you forgot to remove after committing.

Reply:
It's not technically a button. The purpose currently is for it to just be some text that someone can click on. This was the best element for the desired use-case. Regardless, this will be removed/refactored when #109 is completed

Reply:
Okay, ya I didn't realize that it was something that we should click on and thought we should make it look more like a button so people know that it's clickable. But if it's being moved in #109, we can ignore it for now.